programs/server/dashboard.html

```diff
@@ -672,15 +673,39 @@
     chart.addEventListener('mouseleave', e => { edit_buttons.style.display = 'none'; });
 
     charts.appendChild(chart);
+    return {chart: chart, textarea: query_editor_textarea};
 };
 
 document.getElementById('add').addEventListener('click', e => {
     queries.push({ title: '', query: '' });
-    insertChart(plots.length);
+
+    const {chart, textarea} = insertChart(plots.length);
+    chart.scrollIntoView();
+    textarea.focus();
+
     plots.push(null);
     resize();
 });
 
+document.getElementById('reload').addEventListener('click', e => {
+    reloadAll();
+});
+
+function showReloadIfNeeded() {
+    const is_any_field_changed = (host != document.getElementById('url').value
```

Comment:
maybe put condition inside if statement? just style, disregard if you think it is okay...

Reply:
When the condition is complex, I tend to introduce a variable to reflect the meaning of the check in the variable name. But I agree that probably it's not the case. Here the meaning of condition is pretty straightforward. But since it's minor, I'll merge it as is to do it now.